src/python/sdk/examples/trade.py

```diff
@@ -601,7 +702,7 @@ def report_fill_events(fill_events: list[dict]):
     print("LAST ORDER EXECUTION BREAKDOWN: FillEvent(s)")
     if len(fill_events) != 0:
         last_events = find_all_fill_events_with_last_taker_order_id(fill_events)
-        last_events_maker_side = last_events[0]["data"]["maker_side"]
+        last_events_maker_side = last_events[0]["data"]["maker_side"]  # type: ignore
```

Comment:
I find this comment confusing

If we are supposed to ignore it, can this line be delete? What is going on here?

Reply:
This is for the type-checker, which is throwing an error on `last_events[0]` which is spurious. It throws an error anywhere I'm indexing into a list. If I don't include this then I get red squiggly lines.